src/components/provider-list-item/provider-list-item.js

```diff
@@ -11,68 +11,84 @@ import styles from './provider-list-item.css';
 
 const cx = classNames.bind(styles);
 
-function ProviderListItem({ item, link, active, onClick, headingLevel }) {
-  return !item ? (
-    <div className={styles.skeleton} />
-  ) : (
-    <InternalLink
-      data-test-eholdings-provider-list-item
-      to={link}
-      className={cx('item', {
-        'is-selected': active
-      })}
-      onClick={(e) => {
-        if (onClick) {
-          e.preventDefault();
-          onClick();
-        }
-      }}
-    >
-      <Headline
-        data-test-eholdings-provider-list-item-name
-        margin="none"
-        size="medium"
-        tag={headingLevel || 'h3'}
-      >
-        {item.name}
-      </Headline>
-
+const ProviderListItem = ({
+  item,
+  link,
+  active,
+  onClick,
+  headingLevel,
+}) => {
+  return !item
+    ? (
       <div
-        data-test-eholdings-provider-list-item-selections
-        className={cx('itemMetadata')}
+        className={styles.skeleton}
+        data-testid="skeleton-element"
+      />
+    )
+    : (
+      <InternalLink
+        data-test-eholdings-provider-list-item
+        to={link}
+        className={cx('item', {
+          'is-selected': active,
+        })}
+        onClick={(e) => {
+          if (onClick) {
+            e.preventDefault();
+            onClick();
+          }
+        }}
+        data-testid="provider-list-item-link"
       >
-        <SelectedLabel
-          isSelected={item.packagesSelected}
-          selectedCount={item.packagesSelected}
-          showSelectedCount
-        />
+        <Headline
+          data-test-eholdings-provider-list-item-name
+          margin="none"
+          size="medium"
+          tag={headingLevel || 'h3'}
```

Comment:
I think that it's better to add `headingLevel` to default props with value `h3` and you can left here just prop